godtools/App/Share/Views/PageNavigationCollectionView/PageNavigationCollectionView.swift

```diff
@@ -533,12 +533,15 @@ extension PageNavigationCollectionView {
         guard item >= 0 && item < getNumberOfPages() else {
             return false
         }
-                
-        collectionView.scrollToItem(
-            at: IndexPath(item: item, section: 0),
-            at: .centeredHorizontally,
-            animated: animated
-        )
+        
+        DispatchQueue.main.async {
```

Comment:
Hey @levieggertcru, yes this whole scroll method was getting triggered before the collection view was set up, so the scroll wouldn't happen and the collection view just always rendered to the first page.

Reply:
Thanks @rachaelblue.  I'll take a closer look at this.  The renderer navigation is a bit finicky since it has to wait for layout constraints to complete in UIViewController viewDidLayoutSubviews (https://developer.apple.com/documentation/uikit/uiviewcontroller/1621398-viewdidlayoutsubviews).

Reply:
Okay have a fix here (https://github.com/CruGlobal/godtools-swift/pull/2298).  This was broken in all our tools.  I will need to create some navigation tests for initial renderer navigation.